docs/ocis/development/debugging.md

```diff
@@ -179,6 +179,171 @@ For example `deployments/examples/ocis_wopi/docker-compose.yml`
 },
 ```
 
+### Debugging the Collaboration stack
+Sometimes for debugging purposes, we want to run the WOPI stack with the collaboration service locally.
+Let's focus on an example that uses the local runs of the ocis and collaboration together with the containerized traefik and collabora.
+Please note: Against the stack that uses https://github.com/cs3org/wopiserver, we don't need the app_provider anymore. The new collaboration service now includes an app_provider.
+
+
+1. Update the hosts file
+```bash
+127.0.0.1 ocis.owncloud.test
+127.0.0.1 traefik.owncloud.test
+127.0.0.1 collabora.owncloud.test
+127.0.0.1 collaboration.owncloud.test
```

Comment:
The hosts file is not complete, two items need to be added, see https://github.com/owncloud/ocis/pull/9131/files in `ocis_wopi.md` (the reference is the backport) for a complete list, which was added recently.

Reply:
Rest items are redundant in our particular case.